scripts/animation.lua

```diff
@@ -22,19 +18,14 @@ end
 
 
 -- Start smoke effect after the model was built
-function animSmoke(getid, getpiece)
-	local id=getid
-	local piece=getpiece
-	local SMOKE = 257
-	local last_inbt = true
+function animSmoke(id, piece)
+	local SMOKE, last_inbt, inProgress = 257, true
```

Comment:
having them crammed onto a single line hurts readability and `SMOKE` isn't actually used